tests/unit/lib/voice.spec.ts

```diff
@@ -0,0 +1,36 @@
+import { describe, expect, it } from "vitest";
+import { EngineId, SpeakerId, StyleId, Voice, VoiceId } from "@/type/preload";
+
+export const voiceIdToVoice = (voiceId: VoiceId): Voice => {
+  const [engineId, speakerId, styleId] = voiceId.split(":");
+  return {
+    engineId: EngineId(engineId),
+    speakerId: SpeakerId(speakerId),
+    styleId: StyleId(Number(styleId)),
+  };
+};
+
+describe("voiceIdToVoice", () => {
```

Comment:
一番ほしいのは「voiceToVoiceIdのロジックが変更されてないことのテスト」かなと！！

こういうのを想像しています
```suggestion
describe("voiceToVoiceId", () => {
  it("Voice→VoiceId変換処理が変わっていない", () => {
    const voice: Voice = {
      engineId: EngineId("57959e44-0c80-4c21-984d-7ba58224e834"),
      speakerId: SpeakerId("acb2b256-ead4-43b4-b738-84c0f0feb36d"),
      styleId: StyleId(0),
    };

    expect(VoiceId(voice)).toEqual(`${voice.engineId}:${voice.speakerId}:${voice.styleId}`);
  });
}

describe("voiceIdToVoice", () => {
```